src/lib.rs

```diff
@@ -1,30 +1,28 @@
-mod devices;
-mod logging;
-mod backup;
-mod restore;
-mod adb_command;
-mod file_transfer;
-mod database;
-
 extern crate chrono;
-extern crate fern;
-
+extern crate failure;
 #[macro_use]
-extern crate log;
-
+extern crate failure_derive;
+extern crate fern;
 #[cfg(test)]
 #[macro_use]
 extern crate hamcrest;
-
-extern crate failure;
-#[macro_use] extern crate failure_derive;
+#[macro_use]
+extern crate log;
 
 use backup::Backup;
 use database::management::DatabaseManager;
 use devices::Device;
 use failure::{err_msg, Error};
 use restore::Restore;
 
+mod devices;
+mod logging;
+mod backup;
+mod restore;
+mod adb_command;
+mod file_transfer;
+mod database;
+
```

Comment:
might as well sort all `extern crate` `use` and `mod` lines alphabetically while we are at it?